otx/algorithms/segmentation/tasks/inference.py

```diff
@@ -189,7 +189,7 @@ def _init_recipe(self):
         self.model_dir = os.path.abspath(os.path.dirname(self.template_file_path))
         pipeline_path = os.path.abspath(self.data_pipeline_path)
 
-        if train_type not in (TrainType.SEMISUPERVISED, TrainType.INCREMENTAL):
+        if train_type not in (TrainType.SEMISUPERVISED, TrainType.INCREMENTAL, TrainType.SELFSUPERVISED):
```

Comment:
It is minor, but since the number of tasks has increased, how about making (TrainType.SEMISUPERVISED, TrainType.INCREMENTAL, TrainType.SELFSUPERVISED) a global variable for the file?

refer: https://github.com/openvinotoolkit/training_extensions/blob/c076902a791e069cb89abab89550cc13449d6b53/otx/cli/tools/find.py#L30

Reply:
Done.